### ISSUE TITLE
Re-export svelte component

### ISSUE THREAD

**Issue by dishuostec** [tomblachut/svelte-intellij#48]
When re-export multiple svelte components using a js file, webstorm can't recognize component.

![image](https://user-images.githubusercontent.com/384801/62592267-24823980-b905-11e9-8f22-ab73ab2673be.png)

![image](https://user-images.githubusercontent.com/384801/62592241-061c3e00-b905-11e9-9a93-8921d029dc4e.png)

example: https://svelte.dev/repl/e66018598e614b5d874a28fb9fa4ac62?version=3.7.1

**Comment by tomblachut** [comment]
Partially fixed in #39. Imports are still marked as unused :/

**Comment by tomblachut** [comment]
Small update:

```
export {default as Foo} from './Foo.svelte';
export {default as Bar} from './Bar.svelte';
```

works at least in current version. I'm gonna fix this soon™

**Comment by tomblachut** [comment]
![image](https://user-images.githubusercontent.com/615013/90986749-7ab03200-e585-11ea-93b2-1189e5e510ba.png)
will be fixed in v0.14
